Updated alt attributes of images in client side web api in learn/javascript (#19762)
mdn/content: files/en-us/learn/javascript/client-side_web_apis/manipulating_documents/index.md

Despite the limitations, Web APIs still give us access to a lot of functionality that enable us to do a great many things with web pages. There are a few really obvious bits you'll reference regularly in your code — consider the following diagram, which represents the main parts of a browser directly involved in viewing web pages:

> ![Important parts of web browser such as window, navigator, document](document-window-navigator.png)

Reviewer: ```suggestion
![Important parts of web browser; the document is the web page. The window includes the entire document and also the tab. The navigator is the browser, which includes the window (which includes the document) and all other windows.](document-window-navigator.png)
```

The DOM on the other hand looks like this:

> ![Tree structure representation of Document Object Model](dom-screenshot.png)

Reviewer: ```suggestion
![Tree structure representation of Document Object Model: The top node is the doctype and HTML element. Child nodes of the HTML include head and body. Each child element is a branch. All text, even white space, is shown as well.](dom-screenshot.png)
```

The finished demo will look something like this:

> ![Demo layout of a shopping list with input field](shopping-list.png)

Reviewer: ```suggestion
![Demo layout of a shopping list. A 'my shopping list' header followed by 'Enter a new item' with an input field and 'add item' button. The list of already added items is below, each with a corresponding delete button. ](shopping-list.png)
```
